packages/playwright-core/src/outofprocess.ts

```diff
@@ -45,8 +45,7 @@ class PlaywrightClient {
     this._driverProcess.unref();
     this._driverProcess.stderr!.on('data', data => process.stderr.write(data));
 
-    const connection = new Connection(undefined, undefined);
-    connection.markAsRemote();
```

Comment:
Maybe add something like `markAsOutOfProcess` instead of passing encoding format in the constructor in all call sites?

Reply:
I can make it optional as well, just thought implicit is better. So its by default b64 and only for inprocess we need to specify 'buffer'?